packages/client/lib/net/protocol/ethprotocol.ts

```diff
@@ -65,6 +66,17 @@ export class EthProtocol extends Protocol {
     {
       name: 'Transactions',
       code: 0x02,
+      encode: (txs: TypedTransaction[]) => {
+        const serializedTxs = []
+        for (const tx of txs) {
```

Comment:
PR looks good! I'm just taking an initial look and I'm noticing coverage missing for these newly added encode/decode functions. Do you think it would be worth the time to add tests for them? We ran into the same thing with #1443. I think it's important to keep these lines covered to reduce the likelihood of breakage in the future. We should ensure we have a nice test suite that can be expanded easily as more functions are added in these files (ethprotocol and fullethereumservice). I believe there should be existing tests for some of the prior methods that we can just replicate.